Charge/Traits/HasSubscriptions.php

```diff
@@ -190,12 +190,23 @@ public function plan()
     public function plans()
     {
         $limit = $this->getParamInt('limit', 10);
-        $plans = Plan::all(
-            [
-                'expand' => ['data.product'],
-                'limit' => $limit,
-            ]
-        )->toArray();
+        $active = $this->getParam('active');
```

Comment:
Please use `->getParamBool` and remove the bool typecast below

Reply:
getParamBool forces the $active variable to be "false" even if not specified.

If "active=false", Stripe list inactive plans (and not all plans)